### PR TITLE
Fat 2326/c358140

### PR DESCRIPTION
**Pull request by kimkate303**
https://foliotest.testrail.io/index.php?/cases/view/358140

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ import 'cypress-file-upload';
 import './data-import';
 import './commands';
 
-setInteractorTimeout(100_000);
+setInteractorTimeout(10_000);
```
Comment (MaksimTerekhov) [inline cypress/support/index.js:20]: wow, it seems too small value for us)

File: cypress/support/fragments/inventory/search/browseCallNumber.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { Select, TextInput, including, Button, MultiColumnListCell, MultiColumnListRow } from '../../../../../interactors';
+
+const inventorySearch = TextInput({ id: 'input-inventory-search' });
+const browseButton = Button('Browse');
+const row4 = MultiColumnListRow({ index: 4 });
+const row5 = MultiColumnListRow({ index: 5 });
+
+export default {
+  select() {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/search/browseCallNumber.js:9]: already exists, you can find function by name selectBrowseCallNumbers

File: cypress/support/fragments/inventory/search/browseCallNumber.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    // cypress can't draw selected option without wait
+    cy.wait(1000);
+    cy.do(Select('Search field index').choose('Browse call numbers'));
+  },
+
+  browse(callNumber) {
```
Comment (MaksimTerekhov) [inline cypress/support/fragments/inventory/search/browseCallNumber.js:15]: browseSubjectsSearch

Comment (kimkate303) [reply cypress/support/fragments/inventory/search/browseCallNumber.js]: done

File: cypress/support/fragments/inventory/search/browseCallNumber.js (hunk 2 above)
Comment (kimkate303) [reply cypress/support/fragments/inventory/search/browseCallNumber.js]: done

File: cypress/support/index.js (hunk 1 above)
Comment (kimkate303) [reply cypress/support/index.js]: done

**Review by MaksimTerekhov — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**